compiler/qsc_frontend/src/compile/tests.rs

```diff
@@ -962,7 +968,7 @@ fn unimplemented_call_from_dependency_produces_error() {
         [
```

Comment:
This doesn't seem right, but revising `@Unimplemented` relative to exports seems out of scope, especially since it is not used anywhere today (or even documented). Can you either comment out the test attribute (or skip) and then file a follow up issue on me?